oneanddone/tasks/models.py

```diff
@@ -517,6 +524,14 @@ def users_who_completed_this_task(self):
                 users.append(user)
         return users
 
+    @property
+    def total_votes(self):
+       rating= 0
+       for attempt in self.all_attempts:
```

Comment:
You can probably achieve this in less code using django's `aggregate` and `Sum`. Check out https://docs.djangoproject.com/en/1.4/topics/db/aggregation/